<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>410</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="button_get_indices">
       <property name="text">
        <string>Get sobol indices</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_get_S1_conv">
       <property name="text">
        <string>Analyse convergence (S1)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_get_ST_conv">
       <property name="text">
        <string>Analyse convergence (ST)</string>
       </property>
      </widget>
     </item>
      <item>
      <widget class="QPushButton" name="button_export">
       <property name="text">
        <string>Export as .mat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select collection to analyse</string>
    </property>
   </widget>
   <widget class="QComboBox" name="combo_collection">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>581</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>150</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Select attribute to analyse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyse</string>
    </property>
   </widget>
   <widget class="Widget_listWithSearch" name="list_attributes">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>180</y>
      <width>571</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Widget_listWithSearch</class>
   <extends>QWidget</extends>
   <header>optimeed.visualize.widgets.widget_listWithSearch</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
